opendevin/runtime/plugins/agent_skills/setup.sh

```diff
@@ -10,4 +10,4 @@ export PATH=/opendevin/plugins/agent_skills:$PATH
 echo 'export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH' >> ~/.bashrc
 export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH
 
-pip install flake8
+pip install flake8 python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
```

Comment:
Do you think this line is necessary when it has already been added to the docker file?

Reply:
Maybe not? This could be easily verified.

Reply:
Yes - i think we should better keep this. We will want to support when the user brings their own docker image: https://github.com/OpenDevin/OpenDevin/issues/1387

When that happens, we should check whether the dependencies are installed - if not, we should install them.

Also check this PR: https://github.com/OpenDevin/OpenDevin/pull/2101

Reply:
This pip install will just be a `no-op` when it is already installed, so it doesn't hurt i think.

Reply:
Agreed--we want to try and support non-default docker images, so making sure the plugins take care of all their own dependencies is good.

Reply:
`RuntimeError: Failed to initialize plugin jupyter with exit code 1 and output: The folder you are executing pip from can no longer be found.`
Getting weird errors in CI with persistent sandbox.
https://github.com/SmartManoj/Kevin/actions/runs/9330781998/job/25684728084